.github/workflows/deploy-to-azure-app-service.yml

```diff
@@ -247,6 +254,33 @@ jobs:
             -WebAppName ${{ inputs.app-name }} `
             -SlotName ${{ inputs.slot-name }}
 
+      - name: Fetch tags
```

Comment:
Is there any specific reason why these are separate steps, as opposed to having a single script step? It doesn't seem necessary to me, and it introduces `GITHUB_OUTPUT`. Also in the other workflow.